manifests/config.yaml

```diff
@@ -63,3 +63,36 @@ spec:
           when: '{{repl ConfigOptionEquals "postgres_type" "external_postgres_inline"}}'
           type: text
           default: sentry
+
+        - name: redis_type
+          type: select_one
+          title: Redis
+          default: embedded_redis
+          items:
+            - name: embedded_redis
+              title: Embedded Redis
+            - name: external_redis_inline
+              title: External Redis
+        - name: embedded_redis_password
```

Comment:
These is just for config example right? They don't wire in, yet.

Reply:
correct, demo-able but totally vaporware